lib/msal-core/src/error/ClientConfigurationError.ts

```diff
@@ -5,6 +5,10 @@ import { Constants } from "../Constants";
 import { ClientAuthError } from "./ClientAuthError";
 
 export const ClientConfigurationErrorMessage = {
+    configurationNotSet: {
```

Comment:
is this possible? doesnt UAA throw without this

Reply:
More of a sanity check error than anything else.